packages/@storefront/flux-capacitor/test/unit/core/adapters/refinements.ts

```diff
@@ -4,25 +4,77 @@ import Selectors from '../../../../src/core/selectors';
 import suite from '../../_suite';
 
 suite('Refinements Adapter', ({ expect, stub }) => {
+  describe('mergePastPurchaseRefinements', () => {
```

Comment:
If you're only writing one test, it's not necessary to extract variables out to the top

Reply:
Stubs and test data moved into individual cases; marking as resolved.